posts/lte-examples.md

```diff
@@ -0,0 +1,692 @@
+  This is an exceptionally nice abelian category with compact projective generators.
+3. It's possible to interpret any topological abelian group as a condensed abelian group.
+   For example, any $p'$-Banach space $V$, which is a topological vector space over $\mathbb{R}$ satisfying additional conditions, can be viewed as an object of $\mathrm{Cond(Ab)}$.
+   It's also possible to interpret $\mathcal{M}\_{p'}(S)$ (better, any CompHaus-filtered-pseudo-normed-group) as a condensed abelian group. 
+   See below for more details.
+
+The files in the examples folder describe how each of these ingredients has been formalized in the liquid tensor experiment.
+We will discuss each file individually in the following sections.
+
+# The real numbers
+
+A linearly ordered field is called *conditionally complete* provided that every subset which is bounded above has a least upper bound, and every subset which is bounded below has a greatest lower bound.
+The reals are an example of such a field, and any two such fields are (uniquely) isomorphic, as ordered fields.
+
+The file [`examples/real.lean`](https://github.com/leanprover-community/lean-liquid/blob/13777e1f84a324030eed48e636550aee90f8656f/src/examples/real.lean) indicates these facts in the following examples as evidence that Leans' definition of the reals is correct.
+```lean
+-- The reals are a conditionally complete linearly ordered field.
+example : conditionally_complete_linear_ordered_field ℝ := 
+-- the proof ...
+
+-- Any conditionally complete linearly ordered field `K` is 
+-- isomoprphic to the reals.
+example {K : Type*} [conditionally_complete_linear_ordered_field K] : 
+  K ≃+*o ℝ := 
+-- the proof ...
+
+-- If `K` is a conditionally complete linearly ordered field
+-- then any two order-preserving isomorphisms between `K` and 
+-- the reals are equal.
+example {K : Type*} [conditionally_complete_linear_ordered_field K] 
+  (e₁ e₂ : K ≃+*o ℝ) : e₁ = e₂ := 
+-- the proof ...
+```
+In the code above, the symbol `A ≃+*o B` is notation for the type of *isomorphisms of ordered rings* between `A` and `B`, indicating that such an isomorphism is compatible with addition (`+`), multiplication (`*`), and the ordering (`o`). 
+This result is actually a [recent addition](https://github.com/leanprover-community/mathlib/pull/3292) to `mathlib`! 
+
+This file also briefly mentions the following example regarding the nonnegative reals:
+```lean
+example : ℝ≥0 = {r : ℝ // r ≥ 0} :=
+rfl
+```
+ilustrating that `ℝ≥0` is defined to be the collection of all real numbers $r$ satisfying $r \geq 0$.
+This collection of nonnegative reals appreas in the main statement of the challenge, and otherwise used extensively throughout the project.
```

Comment:
```suggestion
This collection of nonnegative reals appears in the main statement of the challenge, and otherwise used extensively throughout the project.
```